components/scream/src/share/tests/kokkos_utils_tests.cpp

```diff
@@ -49,8 +51,8 @@ TEST_CASE("team_policy", "[kokkos_utils]") {
       }
     }
     else {
-#if defined MIMIC_GPU && defined KOKKOS_ENABLE_OPENMP
-      if (omp_get_num_threads() > 1 && p.team_size() == 1) ++nerr;
+#if defined SCREAM_MIMIC_GPU && defined KOKKOS_ENABLE_OPENMP
+      REQUIRE((omp_get_num_threads() == 1 || p.team_size() > 1));
```

Comment:
Not really a big thing, but I think is neater to avoid using raw OpenMP, and rely on Kokkos interface instead. How about `Kokkos::OpenMP::concurrency()`?